main/src/com/google/refine/importers/tree/XmlImportUtilities.java

```diff
@@ -580,40 +575,38 @@ static protected void processSubRecord(
             }
             if (text.length() > 0 | !storeEmptyStrings) {
                 addCell(
-                    project,
-                    thisColumnGroup,
-                    record,
-                    composeName(parser.getAttributePrefix(i), parser.getAttributeLocalName(i)),
-                    text,
-                    storeEmptyStrings,
-                    guessDataType
-                );
+                        project,
+                        thisColumnGroup,
+                        record,
+                        composeName(parser.getAttributePrefix(i), parser.getAttributeLocalName(i)),
+                        text,
+                        storeEmptyStrings,
+                        guessDataType);
             }
         }
 
         while (parser.hasNext()) {
             Token eventType = parser.next();
             if (eventType == Token.StartEntity) {
                 processSubRecord(
-                    project,
-                    parser,
-                    thisColumnGroup,
-                    record,
-                    level+1,
-                    trimStrings,
-                    storeEmptyStrings,
-                    guessDataType
-                );
-            } else if (//eventType == XMLStreamConstants.CDATA ||
-                        eventType == Token.Value) { //XMLStreamConstants.CHARACTERS) {
+                        project,
+                        parser,
+                        thisColumnGroup,
+                        record,
+                        level + 1,
+                        trimStrings,
+                        storeEmptyStrings,
+                        guessDataType);
+            } else if (// eventType == XMLStreamConstants.CDATA ||
+            eventType == Token.Value) { // XMLStreamConstants.CHARACTERS) {
                 Serializable value = parser.getValue();
                 String colName = parser.getFieldName();
                 if (value instanceof String) {
                     String text = (String) value;
-                    if(trimStrings) {
+                    if (trimStrings) {
                         text = text.trim();
                     }
-                    addCell(project, thisColumnGroup, record, colName, text, 
+                    addCell(project, thisColumnGroup, record, colName, text,
```

Comment:
All the changes in this file seem to be formatting changes, is that correct? If so I would prefer to leave this file untouched in this PR. If not, all the formatting changes should be removed so that we can see the functional changes more easily.

Reply:
Am sorting it out

Reply:
I have reverted the file to its original status

Reply:
@wetneb  I have not been able to dicard the formatting changes. Closing this PR and submitting a new one